xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -963,8 +961,18 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   {
     StringUtils::ToLower(strTmp);
     if (strTmp == "horizontal")
+    {
       orientation = HORIZONTAL;
+      // default orientation for a settings slider is vertical
```

Comment:
this does not make much sense to me because we currently don't have any vertical sliders. further it would break all existing skins.